Makefile.in

```diff
@@ -173,6 +173,7 @@ endif
 	@echo
 
 smoke-test: all_nomsg
+	make -C test_regress
```

Comment:
```suggestion
```

use "make test_regress" for the longer tests, and you should have VERILATOR_AUTHOR_SITE=1 in your ~/.bashrc